client/components/app.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      photos: [],
+    };
+  }
+
+  fetchPhoto() {
+    return fetch('/photos/1')
+      .then (function(response) {
+        return response.json();
+      })
+      .then (function(myJson) {
+        return JSON.stringify(myJson);
+      })
+      .catch(err => {if (err) console.log(err)});
+  }
+
+  componentDidMount() {
+    this.fetchPhoto()
+      .then((val) => {
+        this.setState({ photos: val });
+        console.log(val);
+      });
+  }
+
+  render() {
+    return (
+      <div>
+        {this.state.photos}
+      </div>
+    );
+  }
+}
+
+export default App;
```

Comment:
default works with class declarations, so you can move the export default to line 3

Reply:
I don't remember seeing the suggested method before. Is this optional or a recommended practice?

Reply:
Yes, this is a practice recommended by MDN: https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#Description